Main.py

```diff
@@ -306,6 +306,11 @@ def precollect_hint(location: Location, auto_status: HintStatus):
                     game_world.game: worlds.network_data_package["games"][game_world.game]
                     for game_world in multiworld.worlds.values()
                 }
+                dynamic_data_package = {
+                    game_world.game: game_world.get_dynamic_data_package_data()
+                    for game_world in multiworld.worlds.values()
+                    if game_world.get_dynamic_data_package_data() is not None
```

Comment:
You should probably put this in a regular loop instead of calling that method twice just to do a comprehension lol